app-forensics/aflplusplus/files/aflplusplus-4.01c-respect-flags.patch

```diff
@@ -50,6 +50,15 @@
  	ln -sf afl-cc.8 afl-gcc-fast.8
 --- a/GNUmakefile.llvm
 +++ b/GNUmakefile.llvm
+@@ -410,7 +410,7 @@ endif
+ 
+ ./afl-ld-lto: src/afl-ld-lto.c
+ ifeq "$(LLVM_LTO)" "1"
+-	$(CC) $(CFLAGS) $(CPPFLAGS) $< -o $@
++	$(CC) $(CFLAGS) $(CPPFLAGS) $< -o $@ $(LDFLAGS)
```

Comment:
I usually group together:
```suggestion
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) $< -o $@
```
out of habit because of some of the issues with bits like -Wl,--as-needed, but I don't think it actually matters here.